doc/cygnus-ngsi/flume_extensions_catalogue/ngsi_cartodb_sink.md

```diff
@@ -446,8 +448,8 @@ cygnus-ngsi.sinks.cartodb-sink.enable_name_mappings = false
 cygnus-ngsi.sinks.cartodb-sink.enable_lowercase = false
 cygnus-ngsi.sinks.cartodb-sink.keys_conf_file = /usr/cygnus/conf/cartodb_keys.conf
 cygnus-ngsi.sinks.cartodb-sink.swap_coordinates = true
-cygnus-ngsi.sinks.cartodb-sink.enable_raw = true
-cygnus-ngsi.sinks.cartodb-sink.enable_distance = false
+cygnus-ngsi.sinks.cartodb-sink.enable_raw_historic = true
```

Comment:
Yes, everything is within this PR :) Both old and new parameters are supported, nevertheless old parameters warn about they are deprecated when used. In addition, the deprecated functinality document has nen entries for this.